uint8 START = 0
uint8 STOP = 1

uint8 cmd
uint8[] processKeys